app/controllers/application.js

```diff
@@ -1,12 +1,14 @@
 import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
-import { LOGOUT_URL, MAIN_SITE_URL } from '../constants/navbar';
+import { LOGOUT_URL, MAIN_SITE_URL } from '../constants/url';
 import { action } from '@ember/object';
 import { GITHUB_URL } from '../constants/url';
+import ENV from 'website-my/config/environment'; // remove this when new flow goes live
```

Comment:
Can we please remove the comment from here? It was specifically for the signup.js